job-dsls/jobs/downstream_pr_jobs.groovy

```diff
@@ -62,7 +62,8 @@ def final REPO_CONFIGS = [
         "jbpm-designer"             : [],
         "jbpm-wb"                   : [],
         //"kie-docs"                  : [], // no other repo depends on kie-docs
-        //"kie-wb-distributions"      : [] // kie-wb-distributions is the last repo in the chain
+        "kie-wb-distributions"      : [],
+        "jbpm-work-items"           : [] // jbpm-work-items is the last repo in the chain
```

Comment:
Since the `jbpm-work-items` is _not_ part of the `repository-list.txt` (and not if/when that would happen). As such we should include it in the downstream builds. I would revert this change completely - IMO we don't need downstream PR job for `kie-wb-distributions`, nor `jbpm-work-items`.

Reply:
@psiroky I think it is better to close the whole PR and do a new one. But as I commented before - I don't know how to do this well.